lib/tasks/rbs.rake

```diff
@@ -31,7 +31,7 @@ if Rails.env.development? || Rails.env.test?
     end
 
     task :validate do
-      sh "rbs", "-Isig", "validate", "--silent"
```

Comment:
The `--silent` option was deprecated since RBS-3.4.0 silently (refs: https://github.com/ruby/rbs/commit/bc5dc67c87df5158e2204a3e13a55594636fbd32)